sites/funmooc/src/backend/funmooc/settings.py

```diff
@@ -567,6 +568,25 @@ class Base(StyleguideMixin, DRFMixin, RichieCoursesConfigurationMixin, Configura
         },
     }
 
+    # Marketing
+    MARKETING_SITE_ID = values.Value(
+        None, environ_name="MARKETING_SITE_ID", environ_prefix=None
+    )
+
+    MARKETING_LEVELS_2 = values.ListValue(
+        [
```

Comment:
So could this be the result of a Page query something like: 
```python
pages = Page.objects.filter(publisher_is_draft=False, node__depth=1)
```
Note that you could even make a list of the slugs to ease your js investigations of the path?

Reply:
The important detail here is the element order in the dict. I don't think that `Page.objects.filter` can guarantee this order?

Reply:
It can garantee the order of the page in the page tree... Is this what you want?

Reply:
Ok so finally it appears that `level2` are not yet defined in Analytics Suite 2.
So I decided to retrieve the root_node of the current_page and get its primary_key as level2.